lib/mvi/service.rb

```diff
@@ -195,15 +195,15 @@ def message_edipi(user)
       MVI::Messages::FindProfileMessageEdipi.new(user.dslogon_edipi).to_xml
     end
 
-    def message_user_attributes(user)
```

Comment:
now that `find_profile` is being called with a `user_identity`, `create_profile` and  `message_user_attributes` will too.
![Screen Shot 2020-05-13 at 5 01 59 PM](https://user-images.githubusercontent.com/19188/81865164-9003d200-953b-11ea-8338-f6af1499dfe1.png)